From ff6fd5312710c69f7fe3841c2c9ca534260e3394 Mon Sep 17 00:00:00 2001
From: David Mercado <david.mercado@windriver.com>
Date: Wed, 20 Nov 2013 15:34:19 -0500
Subject: [PATCH 145/159] LSI ACP34XX: replace __DEVINIT with __CPUINIT

git.yoctoproject.org/git/linux-yocto-3.10
commit bfd9c6e5ae3589d29bbaa9a4a6713c7880cce432 standard/axxia/base.

__DEVINIT has been removed in linux-3.8.  Replace __DEVINIT with __CPUINIT
to avoid a section mismatch warning in misc_32.S.

Signed-off-by: Hong H. Pham <hong.pham@windriver.com>
---
 arch/powerpc/kernel/misc_32.S |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/kernel/misc_32.S b/arch/powerpc/kernel/misc_32.S
index 0ef9057..a4d937c 100644
--- a/arch/powerpc/kernel/misc_32.S
+++ b/arch/powerpc/kernel/misc_32.S
@@ -675,7 +675,7 @@ _GLOBAL(abs)
 	blr
 
 #ifdef CONFIG_SMP
-__DEVINIT
+__CPUINIT
 _ENTRY(start_secondary_resume)
 	/* Reset stack */
 	CURRENT_THREAD_INFO(r1, r1)
-- 
1.7.5.4

